apps/project/schema.py

```diff
@@ -52,6 +105,24 @@ def resolve_allowed_permissions(root, info) -> List[str]:
             root.get_current_user_role(info.context.request.user)
         )
 
+    @staticmethod
+    def resolve_stats(root, info, **kwargs):
+        return info.context.dl.project.project_stat.load(root.pk)
+
+    @staticmethod
+    def resolve_activity_log(root, info, **kwargs):
+        return list(project_activity_log(root))
+
+    @staticmethod
+    def resolve_top_sourcers(root, info, **kwargs):
+        from lead.models import Lead
+        return get_top_entity_contributor(root, Lead)
+
+    @staticmethod
+    def resolve_top_taggers(root, info, **kwargs):
+        from entry.models import Entry
+        return get_top_entity_contributor(root, Entry)
```

Comment:
Use dataloader when there is n+1.

Reply:
Moving this to `ProjectDetailType` and using that in `recent_projects`